README.md

```diff
@@ -57,18 +57,18 @@ Currently there are several phases, representing an order of projects, which we
 * Phase 1:
   * [che-code](https://github.com/che-incubator/che-code), 
   * [che-machine-exec](https://github.com/eclipse-che/che-machine-exec), 
-  * [che-dashboard](https://github.com/eclipse-che/che-dashboard), 
```

Comment:
why delete dashboard?

Reply:
I think Dashboard depends on devworkspace-generator so it was moved to Phase 2